src/DataShare/DataShare/Account/GetAzDataShareAccount.cs

```diff
@@ -0,0 +1,143 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+namespace Microsoft.Azure.Commands.DataShare.Account
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Management.Automation;
+    using System.Net;
+    using Microsoft.Azure.Commands.DataShare.Common;
+    using Microsoft.Azure.Commands.DataShare.Helpers;
+    using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+    using Microsoft.Azure.Management.DataShare;
+    using Microsoft.Azure.Management.DataShare.Models;
+    using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+    using Microsoft.Azure.PowerShell.Cmdlets.DataShare.Extensions;
+    using Microsoft.Azure.PowerShell.Cmdlets.DataShare.Models;
+    using Microsoft.Azure.PowerShell.Cmdlets.DataShare.Properties;
+    using Microsoft.Rest.Azure;
+
+    /// <summary>
+    /// Defines the Get-AzDataShareAccount cmdlet.
+    /// </summary>
+    [Cmdlet(
+         "Get",
+         ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "DataShareAccount",
+         DefaultParameterSetName = ParameterSetNames.FieldsParameterSet,
+         SupportsPaging = true),
```

Comment:
@gaurangisaxena we should remove the `SupportsPaging` property from this cmdlet attribute since it will add three additional parameters in the background: `-First`, `-Top` and `-IncludeCount`. These three don't seem to be supported in this cmdlet currently, so it would be best if we just removed this property so those parameters aren't added by default.